src/Row.tsx

```diff
@@ -33,6 +34,9 @@ function Row<R, SR>(
   }: RenderRowProps<R, SR>,
   ref: React.Ref<HTMLDivElement>
 ) {
+  const defaultComponents = useDefaultRenderers<R, SR>();
+  const cellRenderer = defaultComponents?.renderCell ?? defaultRenderCell;
```

Comment:
```suggestion
  const renderCell = useDefaultRenderers<R, SR>().renderCell!;
```
This should not be undefined here